docker/config/devnet/tendermint/mnemonics.txt

```diff
@@ -0,0 +1 @@
+ordinary mandate edit father snack mesh history identify print borrow skate unhappy cattle tiny first
```

Comment:
Is it ok to include these config files to repository? 
Is this tool supposed to be used by developers or users?

Reply:
this just for develop, i add mnemonics.txt here so that we can `wallet restore` it and have some test.

Reply:
It would be good to use chainbot to generate these config files on the fly, the only downside is we need to add python3 as prerequisite.

Reply:
yea, we can combine the two tools, but first I made it works and the simpler the better, we can optimize later.